package.json

```diff
@@ -25,36 +24,31 @@
     "url": "https://github.com/electron/asar/issues"
   },
   "scripts": {
+    "build": "tsc",
```

Comment:
```suggestion
    "build": "rimraf ./lib && tsc",
```

Reply:
I don't think this is correct, it would break incremental tsc

Reply:
Fair point - we should enable incremental compilation though, it's off by default